src/decorators/Mention/addMention.js

```diff
@@ -28,6 +29,9 @@ export default function addMention(
   if (selectedBlockText[focusOffset] === ' ') {
     spaceAlreadyPresent = true;
   }
+  if(mouseSelect != undefined){
```

Comment:
Please replace this condition with `if(mouseSelect){}`

Reply:
the change has been made